rails/init.rb

```diff
@@ -1,16 +1,16 @@
 require "pagem"
 
-['/public/stylesheets/pagem.css'].each do |file|
-    source = File.join(directory,file)
-    dest = RAILS_ROOT + file
-    FileUtils.cp(source, dest)
-end
-
-
-# move images to public/images/medidata_buttons
-['/public/images/pagem'].each do |dir|
-  source = File.join(directory,dir)
-  dest = RAILS_ROOT + dir
-  FileUtils.mkdir_p(dest)
-  FileUtils.cp(Dir.glob(source+'/*.*'), dest)
-end
\ No newline at end of file
+# ['/vendor/assets/stylesheets/pagem.css'].each do |file|
+#     source = File.join(directory,file)
+#     dest = RAILS_ROOT + file
+#     FileUtils.cp(source, dest)
+# end
+#
+#
+# # move images to vendor/assets/images/medidata_buttons
+# ['/vendor/assets/images/pagem'].each do |dir|
+#   source = File.join(directory,dir)
+#   dest = RAILS_ROOT + dir
+#   FileUtils.mkdir_p(dest)
+#   FileUtils.cp(Dir.glob(source+'/*.*'), dest)
+# end
```

Comment:
Do we need to comment these out? This actually copies these assets from this gem into the Rails project the gem is being used in. You won't see issues in your existing local copy or on sandbox, but if you clone a fresh Balance into a new directory or launch a new node in Medistrano, the assets wouldn't be properly copied.